### DISCUSSION TITLE
May I train dp using mixed datasets with different number of elements

### DISCUSSION THREAD

**Discussion by phantaum** [deepmodeling/deepmd-kit#1653] [Q&A] [answered]
Dear all,
If I have two datasets, one contains molecules made by C and H, another does some subsititution of C or H to O, thus made by C, H, O. May I use these two datasets together to train potentials for C,H,O?
Thanks.

**Comment by njzjz** [comment, marked as answer]
Yes.

**Reply by hujianyuxhl** [reply]
so how to write the key word "type' in input.json , such as, "C,H,O"? but just "C,H"in data.set

**Reply by njzjz** [reply]
You can write C,H,O when there are only C,H in the dataset.

**Comment by likefallwind** [comment]
I think C and H means, C and H and 0% O

**Reply by phantaum** [reply]
yes
